react/react1/week1/static-todo-list/src/components/Title.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const Title = () => {
+  return <p>Todo list</p>;
```

Comment:
If you're using the arrow function then keep using it everywhere. Your App.js is written differently.